docs/developer/README.md

```diff
@@ -34,6 +34,26 @@ DJANGO_SECRET_LOGIN_KEY="<...>"
 
 You'll need to create the `.env` file yourself. Get the secrets from Cloud.gov by running `cf env getgov-YOURSANDBOX`. More information is available in [rotate_application_secrets.md](../operations/runbooks/rotate_application_secrets.md).
 
+## Adding user to /admin
+
+To be able to view and use /admin locally:
+
+1. Login via login.gov
+2. Go to the home page and make sure you can see the part where you can submit an application
+3. Go to /admin and it will tell you that UUID is not authorized, copy that UUID for use in 4
+4. in src/registrar/fixtures.py add to the ADMINS list in that file by adding your UUID as your username along with your first and last name. See below:
+
+```
+ ADMINS = [
+        {
+            "username": "<UUID here>",
+            "first_name": "",
+            "last_name": "",
+        },
+        ...
+ ]
+```
+
```

Comment:
Do you want to mention once it has been added to this, it will also create "fake domains" along with various statuses as well. This is a good way to verify that its working.

Reply:
I didn't realize adding yourself to /admins is what triggered that, I assumed that was set elsewhere. Yes I will add that in.

Reply:
updated let me know if there is anything else that should be changed